contrib/kafka/filters/network/test/broker/integration_test/BUILD

```diff
@@ -9,21 +9,21 @@ licenses(["notice"])  # Apache 2
 
 envoy_contrib_package()
 
+# This test sets up multiple services, and this can take variable amount of time (30-60 seconds).
 envoy_py_test(
     name = "kafka_broker_integration_test",
     srcs = [
         "kafka_broker_integration_test.py",
         "@kafka_python_client//:all",
     ],
     data = [
-        "//source/exe:envoy-static",
+        "//contrib/exe:envoy-static",
         "//bazel:remote_jdk11",
         "@kafka_server_binary//:all",
     ] + glob(["*.j2"]),
     flaky = True,
     python_version = "PY3",
     srcs_version = "PY3",
-    tags = ["manual"],
```

Comment:
Given that Kafka has moved to contrib, why not give this a try, so far it appears to be stable.
In the worst case we'd go back to manual mode (and we're still keeping flaky just in case of slow machines, but the default of `moderate` = 5 minutes should be a good first idea).